tests/integration/template_weekly_legacy_3.0.0_bundles.cfg

```diff
@@ -0,0 +1,168 @@
+# Test bundles.
```

Comment:
Legacy `cfg` files were copied from `main` as of `v3.0.0`'s release. These ensure there are no backwards compatibility issues. That is, they make sure nothing _breaks_. Functionality, however, may shift. For instance, setting `make_viewer=False` (the default) now ignores the component plots (see https://github.com/E3SM-Project/zppy/pull/722#issuecomment-2963770032).